docs/ops/activation/SoftPlus_4.md

```diff
@@ -0,0 +1,45 @@
+## SoftPlus <a name="SoftPlus"></a>
+
+**Versioned name**: *SoftPlus-4*
+
+**Category**: *Activation*
+
+**Short description**: SoftPlus takes one input tensor and produces output tensor where the softplus function is applied to the tensor elementwise.
+
+**Detailed description**: For each element from the input tensor calculates corresponding
+element in the output tensor with the following formula: `features[i] = log(exp(features[i]) + 1.0)`
```

Comment:
Rewrite the mathematical function like in https://github.com/openvinotoolkit/openvino/pull/1149

Reply:
Done.